tasks/kb0337e_extend_ca_cert.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source "$PT__installdir/support_tasks/files/common.sh"
+
+echo "test" | base64 -w 0 - &>/dev/null || fail "base64_test" "This utility requires a version of base64 with the -w flag"
+
+new_cert="$(bash "$PT__installdir/support_tasks/files/extend.sh")" || fail "extend_cert"
+contents="$(base64 -w 0 $new_cert)" || fail "encode_cert"
```

Comment:
~~~
contents="$(base64 -w 0 $new_cert)" || fail "encode_cert"
                        ^-- SC2086: Double quote to prevent globbing and word splitting.
~~~

Reply:
Fixed.

Reply:
You can test this locally by running the rake jobs before submitting commit. 

~~~
export SHELLCHECK_OPTS="-e SC1117"
bundle install
bundle exec rake shellcheck tasknames
bundle exec rake check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop syntax lint metadata_lint
~~~

Reply:
and some of the ones it flags that we cant work with because of how tasks and plans work, we can add to the -e switch